css/main.css

```diff
@@ -371,3 +375,39 @@ div.loading {
 	0% { transform: rotate(0deg); }
 	100% { transform: rotate(360deg);}
 }
+
+
+.btn-famnm {
+	background-color: #ffd536;
+	color: black;
+}
+
+.btn-famnm:hover {
+	background-color: #ffe582;
+}
+
+.btn-famnm-secondary {
+	background-color: #3b8ac3;
+	color: white;
+}
+
+.btn-famnm-secondary:hover {
+	background-color: #59a9e3;
+	color: white;
+}
```

Comment:
{nit} Should we be using the FAMNM white from the color palette instead of the box standard one?

Reply:
good call; i'll replace white with the light grey in the color palette.